### PR TITLE
Add loadbalancer settings to service.yaml

### PR DESCRIPTION
**Pull request by YuikoTakada**
This PR adds loadbalancer settings to service.yaml

### CODE REVIEW

File: charts/mattermost-team-edition/templates/service.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 apiVersion: v1
 kind: Service
 metadata:
+{{- if .Values.service.annotations }}
+  annotations:
+{{ toYaml .Values.service.annotations | indent 4 }}
```
Comment (cpanato) [inline charts/mattermost-team-edition/templates/service.yaml]: can you add this to the values.yaml and document?

File: charts/mattermost-team-edition/templates/service.yaml [hunk 2]
```diff
@@ -18,3 +22,7 @@ spec:
     targetPort: http
     protocol: TCP
     name: {{  include "mattermost-team-edition.name" . }}
+{{- if .Values.service.loadBalancerSourceRanges }}
+  loadBalancerSourceRanges:
+{{ toYaml .Values.service.loadBalancerSourceRanges | indent 4 }}
```
Comment (cpanato) [inline charts/mattermost-team-edition/templates/service.yaml]: same here

**Comment by YuikoTakada** [comment]
@cpanato thank you for your comments. I fixed them, so please take a look.

File: charts/mattermost-team-edition/templates/service.yaml (hunk 1 above)
Comment (YuikoTakada) [reply charts/mattermost-team-edition/templates/service.yaml]: done

File: charts/mattermost-team-edition/templates/service.yaml (hunk 2 above)
Comment (YuikoTakada) [reply charts/mattermost-team-edition/templates/service.yaml]: done

**Comment by jasonblais** [comment]
Hey @cpanato, friendly reminder to help with reviews when you have a chance :)

**Comment by YuikoTakada** [comment]
@cpanato thank you for your comments. I've fixed where you gave me comments, so please take a look.
@jasonblais thanks!

**Pull request merged**